src/dom/Null.ts

```diff
@@ -41,10 +61,72 @@ export class Null extends DomValue(Object, IonTypes.NULL) {
         return true;
     }
 
+    // If a [type]Value() operation was called on this Null value, we need to see whether the associated Ion type
+    // supports that conversion. If it does, we'll return a JS null. If it doesn't, we'll throw an Error.
+    private _convertToJsNull(operation: string): null | never {
+        if (Null._operationIsSupported(this.getType(), operation)) {
+            return null;
+        }
+        throw new Error(`'${operation}' is not supported by Ion type ${this.getType().name}`);
+    }
+
+    // If this Null's Ion type supports the requested operation, throw an Error indicating this was a null dereference.
+    // Otherwise, throw an Error indicating that the requested operation is not supported.
+    private _unsupportedOperationOrNullDerefence(operation: string): never {
+        if (Null._operationIsSupported(this.getType(), operation)) {
+            throw new Error(`${operation} called on a null ${this.getType().name}.`);
```

Comment:
add `()`, as in:  `${operation}() called...`

Reply:
Done.